restapi/src/main/java/io/stargate/web/impl/RestApiServer.java

```diff
@@ -170,16 +172,18 @@ protected void configure() {
 
     if (!enableSgv1) {
```

Comment:
Do we document this behavior anywhere? Should be in the README?

Reply:
Ok. Yes, I will update this in a bit. Even if it's transient, makes sense.

Reply:
@jeffreyscarpenter Ok, added a note on opt-in property. I hope this now works better?